ansible/playbooks/aix.yml

```diff
@@ -465,78 +465,132 @@
+        - name: TestIBM XLC13
           command: /opt/IBM/xlC/13.1.3/bin/xlc -qversion
-          register: xlc_qversion
-          tags: xlc
+          register: xlc13_qversion
+          tags: xlc13
 
-        - name: Display XL C qversion information
+        - name: Display XLC13 qversion information
           debug:
-            msg: "{{ xlc_qversion.stdout }}"
-          tags: xlc
+            msg: "{{ xlc13_qversion.stdout }}"
+          tags: xlc13
 
         - name: Symlink - /usr/bin/xlc
           file: src=/opt/IBM/xlC/13.1.3/bin/xlc dest=/usr/bin/xlc state=link
-          when: xlc.stat.islnk is not defined
-          tags: xlc
+          when: xlc13.stat.islnk is not defined
+          tags: xlc13
 
         - name: Symlink - /usr/bin/xlc++
           file: src=/opt/IBM/xlC/13.1.3/bin/xlc++ dest=/usr/bin/xlc++ state=link
-          when: xlc.stat.islnk is not defined
-          tags: xlc
+          when: xlc13.stat.islnk is not defined
+          tags: xlc13
 
         - name: Symlink - /usr/bin/xlC
           file: src=/opt/IBM/xlC/13.1.3/bin/xlC dest=/usr/bin/xlC state=link
-          when: xlc.stat.islnk is not defined
-          tags: xlc
+          when: xlc13.stat.islnk is not defined
+          tags: xlc13
 
         - name: Symlink - /usr/bin/xlc_r
           file: src=/opt/IBM/xlC/13.1.3/bin/xlc_r dest=/usr/bin/xlc_r state=link
-          when: xlc.stat.islnk is not defined
-          tags: xlc
+          when: xlc13.stat.islnk is not defined
+          tags: xlc13
 
         - name: Symlink - /usr/bin/xlC_r
           file: src=/opt/IBM/xlC/13.1.3/bin/xlc_r dest=/usr/bin/xlC_r state=link
-          when: xlc.stat.islnk is not defined
-          tags: xlc
+          when: xlc13.stat.islnk is not defined
+          tags: xlc13
 
         - name: Symlink - /usr/bin/gxlC
           file: src=/opt/IBM/xlC/13.1.3/bin/gxlC dest=/usr/bin/gxlC state=link
-          when: xlc.stat.islnk is not defined
-          tags: xlc
+          when: xlc13.stat.islnk is not defined
+          tags: xlc13
+
+        - name: Clean XLC13 tmp files
+          file:
+            path: /tmp/usr
+            state: absent
+          tags: xlc13
+
+      ###################
+      # IBM XL C 16.1.0 #
+      ###################
+        - name: Checking for XLC16 availability
+          stat:
+            path: /opt/IBM/xlc/16.1.0/bin/xlc
+          register: xlc16
+          tags: xlc16
+
+        - debug:
+            msg: "xlc16 installed, skipping download and installation"
+          when: xlc16.stat.islnk is defined
+          tags: xlc16
+
+        - name: Transfer and Extract XLC16
+          unarchive:
+            src: /Vendor_Files/aix/XLC/XL_C_Cpp_FOR_AIX_V16.1_EMG.tar.Z
+            dest: /tmp
+            remote_src: no
+          when: xlc16.stat.islnk is not defined
+          tags: xlc16
+
+        - name: Install IBM XLC16 - installp
+          command: installp -aXYgd -e /tmp/usr/install.log /tmp/usr/sys/inst.images all
```

Comment:
`/tmp/usr/sys/inst.images` is the install images location directory which is specified with `-d` options.
 Move  `/tmp/usr/sys/inst.images` after `-aXYgd`:  e.g. `installp -aXYgd /tmp/usr/sys/inst.images -e /tmp/usr/install.log all` or 
move the `d` option: e.g. `installp -aXY -e /tmp/usr/install.log  -d /tmp/usr/sys/inst.images all`

Reply:
Of course.  PR updated.